Automattic/wp-calypso
# Emails: Fix billing interval selector in Email Home

#### Changes proposed in this Pull Request

The billing interval selector is not working when url is like:
`https://wordpress.com/email/{domain}/purchase/{site-slug}`

#### Testing instructions

1. Have a domain without email subscription
2. Go to Upgrades
3. Go to Domains
4. Select that domain
5. Click on the CTA in the right side of the screen prompting to add a new Professional Email
6. The billing interval should work and modify the price of the products.

Related to {1200182182542585-as-1202048791217778}
